spec/system/comparisons/heating_in_warm_weather_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+describe 'heating_in_warm_weather' do
+  let!(:school) { create(:school) }
+  let(:key) { :heating_in_warm_weather }
+  let(:advice_page_key) { :heating_control }
+
+  let(:gas_variables) do
```

Comment:
I'd put these in the before block since they don't need to be lets but personal style I guess